src/vaults/__snapshots__/harvests.utils.spec.ts.snap

```diff
@@ -237,6 +255,7 @@ Array [
     "grossApr": 10.650635139472206,
     "timestamp": 1651437281000,
     "token": "0x62B9c7356A2Dc64a1969e19C23e4f579F9810Aa7",
+    "tx": undefined,
```

Comment:
should these be undefined?

Reply:
not, but would be easier to fix, after mocks.pkg upd, forgot to do that

Reply:
can do as a follow up